### ISSUE TITLE
Protection for function overwriting would be great

### ISSUE THREAD

**Issue by msageryd** [TablePlus/TablePlus#472]
Postgres 10
TablePlus 110

I often take an existing function as a template when creating a new similar function. The ctrl-S thing is very convenient when I want to alter an existing function, but this has also bitten me.

1. Open an existing function by clicking on it in the sidebar
2. Change the name and the body to your suiting
3. Accidentally hit Ctrl-S (instead och copying the new function to an SQL tab to create the new function)
4. You have now overwritten the original function in spite of the name change

Solution:
1. Either don't allow a function with a new name overwrite an existing function
2. Or, simply create a new function with the new name (preferable)

**Comment by msageryd** [comment]
Update..
The function was not overwritten. After refreshing I found that the original function still was intact.

But:
- What happened to the new function? Lost in space?
- It would still be great with a solution a la point 2 above.

**Comment by huyphams** [comment]
Yup, the function will be overwritten because TablePlus will drop the old function first. You can create a new function by right click on the item listing: `New > Function`.
